File: 1-Operadores/ex1.php
<?php
echo "== Operadores Aritméticos ==\n";

echo "Digite o primeiro número: ";
$num1 = trim(fgets(STDIN));

echo "Digite o segundo número: ";
$num2 = trim(fgets(STDIN));

$num1 = (float)$num1;
$num2 = (float)$num2;

$soma = $num1 + $num2;
$subtracao = $num1 - $num2;
$multiplicacao = $num1 * $num2;
$divisao = ($num2 != 0) ? $num1 / $num2 : "Indefinida (divisão por zero)";
$resto = ($num2 != 0) ? $num1 % $num2 : "Indefinido (divisão por zero)";
$potencia = $num1 ** $num2;

echo "A soma dos números foi: $soma\n";
echo "A subtração do primeiro pelo segundo é: $subtracao\n";
echo "A multiplicação dos números é: $multiplicacao\n";
echo "A divisão do primeiro pelo segundo é: $divisao\n";
echo "O resto da divisão do primeiro pelo segundo é: $resto\n";
echo "$num1 elevado a $num2 é: $potencia\n";
?>

File: 1-Operadores/ex1.py
print("==Operadores Calculaveis==")
print()
num1 = int(input("Insira um numero: "))
num2 = int(input("Insira outro numero: "))
print("Operações disponiveis")
print("soma | subtracao | multiplicacao | divisao | resto | potencia")
decisao = input("Como deseja calcular? ")

if decisao == "soma":
    resultado = num1 + num2
    print("A soma dos números foi", resultado)

elif decisao == "subtracao":
    resultado = num1 - num2
    print("A subtração dos números foi", resultado)

elif decisao == "multiplicacao":
    resultado = num1 * num2
    print("A multiplicação dos números foi", resultado)

elif decisao == "divisao":
    if num2 != 0:
        resultado = num1 / num2
        print("A divisão dos números foi", resultado)
    else:
        print("Erro: divisão por zero não permitida.")

elif decisao == "resto":
    if num2 != 0:
        resultado = num1 % num2
        print("O resto da divisão é", resultado)
    else:
        print("Erro: divisão por zero não é permitida.")

elif decisao == "potencia":
    resultado = num1 ** num2
    print(f"{num1} elevado a {num2} é: {resultado}")

else:
    print("Operação inválida.")
    